### PR TITLE
Fixes #18153: rudder-api-client install a non fonctionning rudder-cli

### PR DESCRIPTION
**Pull request by ncharles**
https://issues.rudder.io/issues/18153

### CODE REVIEW

File: cli/rudder-cli [hunk 1]
```diff
@@ -32,9 +34,10 @@ import requests
 import json
 import os
 import re
-import sys
 
-requests.packages.urllib3.disable_warnings(requests.packages.urllib3.exceptions.InsecureRequestWarning)
+import urllib3
+urllib3.disable_warnings()
```
Comment (peckpeck) [inline cli/rudder-cli:39]: You have to do both in a try catch because request can use either one depending on situaion

Comment (ncharles) [reply cli/rudder-cli:39]: we do only the later in ncf, i figured it would be safe

**Comment by ncharles** [comment]
PR updated with a new commit

File: cli/rudder-cli (hunk 1 above)
Comment (peckpeck) [reply cli/rudder-cli]: that's why we still have many unsecure request warnings log on some servers

**Review by VinceMacBuche — APPROVED** [review]

**Pull request merged**